Tests iOS/MyList/ArchiveFiltersTests.swift

```diff
@@ -0,0 +1,81 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import XCTest
+import Sails
+import Combine
+import NIO
+
+
+class ArchiveFiltersTests: XCTestCase {
+    var server: Application!
+    var app: PocketAppElement!
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+
+        let uiApp = XCUIApplication()
+        app = PocketAppElement(app: uiApp)
+
+        server = Application()
+
+        server.routes.post("/graphql") { request, _ in
+            let apiRequest = ClientAPIRequest(request)
+
+            if apiRequest.isForSlateLineup {
```

Comment:
I could see ClientAPIRequest having a `requestType` (or some such) property that returns an enum. Then we could turn these `if else` statements into a `switch`. Not necessary now, but food for thought